third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompt-dialog-main.js

```diff
@@ -109,6 +109,14 @@ function closeWindowClickListener() {
   window.close();
 }
 
+function removeCloseButton() {
+  var closeButton = document.getElementById('close-window');
+  closeButton.remove();
```

Comment:
If we really remove the element here, wouldn't the `goog.events.listen()` call in line 125 crash?
Either we need to call `removeCloseButton()` after adding all listeners, or we should do `.style.display = 'none'` instead of `remove()`.